### PR TITLE
Added some further information to readme

### PR DESCRIPTION
**Pull request by rpapallas**

### CODE REVIEW

File: cob_openni2_tracker/launch/body_tracker_nodelet.launch [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 
 	<node pkg="nodelet" type="nodelet" name="body_tracker" args="load cob_openni2_tracker/body_tracker_nodelet /$(arg camera_name)/$(arg camera_name)_nodelet_manager" output="screen" cwd="ROS_HOME">
-		<param name="camera_frame_id" type="str" value="$(arg camera_name)_link" /> <!-- $(arg camera_name)_depth_optical_frame -->
+		<param name="camera_frame_id" type="str" value="$(arg camera_name)_rgb_frame" />   <!-- "$(arg camera_name)_link" -->
```
Comment (ipa-rmb) [inline cob_openni2_tracker/launch/body_tracker_nodelet.launch:24]: Why did you change this parameter? Was this on purpose? Otherwise I need to check tomorrow which setting was correct.

**Review by ipa-rmb — COMMENTED** [review]
Thanks for your contributions in improving the setup and usage instructions. Can you please answer my comment regading the launch file, then I will merge your pull request.

File: cob_openni2_tracker/launch/body_tracker_nodelet.launch (hunk 1 above)
Comment (rpapallas) [reply cob_openni2_tracker/launch/body_tracker_nodelet.launch:24]: Hi.

That is strange, I can't recall if I made the change, if is my change then probably either was some debugging I did to find out a way to get the tf frames or by mistake.

You can safely ignore this change.

Comment (rpapallas) [reply cob_openni2_tracker/launch/body_tracker_nodelet.launch:24]: @ipa-rmb Btw if you see this commit is coming from ipa-rmb committed on 3 Aug 2016 but I don't know why is committed in my changes.

Comment (ipa-rmb) [reply cob_openni2_tracker/launch/body_tracker_nodelet.launch:24]: Hehe yes, you are right. That came from my branch, but it is not yet correct. Anyways, I will accept your pull request and then upload my correction to the coordinate system definitions. Please update your software after one hour with my current status. By the way, are you using a Kinect or Asus device? I assume that there is a difference in coordinate systems between Kinect and Asus, but I do not have a Kinect at hand to test this. So maybe we now have wrong coordinate systems with a Kinect after my update.

Comment (rpapallas) [reply cob_openni2_tracker/launch/body_tracker_nodelet.launch:24]: I am currently working with ASUS. However, we are interested testing this with Microsoft Kinect v2 (Xbox One) which we have in hand. We are planning to test this since ASUS's sensor is discontinued and we want to have latest sensor working with the project. 

So is Kinect v2 supposed to work with this package? If so, we can try and let you guys know if it does, but keep in mind will take a while since we have other priorities over this for now.

**Comment by ipa-rmb** [comment]
You might go ahead and try the software update.
